docs/api.rst

```diff
@@ -726,6 +726,12 @@ section generates a new :class:`URL` instance.
       Support subclasses of :class:`int` (except :class:`bool`) and :class:`float`
       as a query parameter value.
 
+.. method:: URL.without_query_keys(*keys)
```

Comment:
I think the official term is a "query parameter". Can we use it in the API and texts, please?